.github/workflows/ci.yaml

```diff
@@ -88,6 +88,7 @@ jobs:
         with:
           github_token: ${{ secrets.github_token }}
           locale: "US"
```

Comment:
According to `misspell` [docs](https://github.com/golangci/misspell?tab=readme-ov-file#usage), you could remove the `locale` param and that would use a 'neutral' variant of english:
> Correct spellings using locale preferences for US or UK.  Default is to use a neutral variety of English.  Setting locale to US will correct the British spelling of 'colour' to 'color'